@stellar/anchor-tests/src/tests/sep12/getCustomer.ts

```diff
@@ -187,12 +209,32 @@ export const canFetchExistingCustomerById: Test = {
     },
     ...genericFailures,
   },
-  async run(_config: Config): Promise<Result> {
+  async run(config: Config): Promise<Result> {
+    if (
+      !config.sepConfig ||
+      !config.sepConfig["12"] ||
+      !config.sepConfig["12"].customers ||
+      !config.sepConfig["12"].createCustomer ||
+      !config.sepConfig["12"].customers[config.sepConfig["12"].createCustomer]
```

Comment:
since you're checking for `createCustomer` in a config a few times, it might be helpful to have a helper method that does this check so you don't have to write out the check every time:
```
const hasCustomer = (config):Boolean => !config?.sepConfig?.["12"]?.customers?.[config?.sepConfig?.["12"]?.createCustomer]
```

Reply:
this is more NIT, up to you if you want to add this